### ISSUE TITLE
Treeverse broken in Chrome 79.0.3945.88

### ISSUE THREAD

**Issue by paulgb** [paulgb/Treeverse#52]
Reported on Twitter: https://twitter.com/ozakiiiiiiiiiii/status/1207324851989549056

**Comment by paulgb** [comment]
This is the error message:

    Cross-Origin Read Blocking (CORB) blocked cross-origin response https://api.twitter.com/2/timeline/conversation/1207324851989549056.json?include_reply_count=1&tweet_mode=extended with MIME type application/json. See https://www.chromestatus.com/feature/5629709824032768 for more details.

An explanation specific to Chrome Extensions is here: https://www.chromium.org/Home/chromium-security/extension-content-script-fetches/

tl;dr fetches have to be made from a background page instead of the content script. I'm worried this move will clash with other security measures in place, but I'll give it a shot.

**Comment by paulgb** [comment]
Deployed in Treeverse 3.4 for Chrome. As a follow-up, need to make the updates work in Firefox.
